src/NuGet.Clients/NuGet.CommandLine/Commands/SpecCommand.cs

```diff
@@ -38,7 +38,7 @@ public bool Force
         public override void ExecuteCommand()
         {
             string sampleProjectUrl = "http://project_url_here_or_delete_this_line/";
-            string sampleIconUrl = "http://icon_url_here_or_delete_this_line/";
+            string sampleIconFile = "$icon_file_name_within_the_package_here_or_delete_this_line$";
```

Comment:
```suggestion
            string sampleIconFile = "icon.png";
```

NuGet uses `$something$` as a variable that gets replaced when packing a (legacy) csproj. There is no msbuild property, environment variable for assembly attribute "icon_file_name_within_the_package_here_or_delete_this_line", so I wouldn't use this same token replacement syntax. Especially since it's commented out now.

Reply:
Ok, addressed.